osvr/RenderKit/RenderManagerOpenGL.cpp

```diff
@@ -795,43 +841,19 @@ namespace renderkit {
         }
         checkForGLError("RenderManagerOpenGL::RenderDisplayInitialize end");
 
-		// Store the frame buffer that was active before we started rendering,
-		// so we can put it back when we finalize.
-		GLint fb;
-		glGetIntegerv(GL_FRAMEBUFFER_BINDING, &fb);
-		m_initialFrameBuffer = static_cast<GLuint>(fb);
-
 		return true;
     }
 
     bool RenderManagerOpenGL::RenderEyeInitialize(size_t eye) {
         checkForGLError("RenderManagerOpenGL::RenderEyeInitialize starting");
 
         // Render to our framebuffer
-        glBindFramebuffer(GL_FRAMEBUFFER, m_frameBuffers.at(GetDisplayUsedByEye(eye)));
+        glBindFramebuffer(GL_FRAMEBUFFER, m_frameBuffers.at(eye));
         if (checkForGLError(
                 "RenderManagerOpenGL::RenderEyeInitialize glBindFrameBuffer")) {
             return false;
         }
 
-        // Set color and depth buffers for the frame buffer
-        glFramebufferTexture2D(GL_FRAMEBUFFER, GL_COLOR_ATTACHMENT0, GL_TEXTURE_2D,
-                             m_colorBuffers[eye].OpenGL->colorBufferName, 0);
-        glFramebufferRenderbuffer(GL_FRAMEBUFFER, GL_DEPTH_ATTACHMENT,
-                                  GL_RENDERBUFFER, m_depthBuffers[eye]);
-        if (checkForGLError(
-                "RenderManagerOpenGL::RenderEyeInitialize Setting textures")) {
-            return false;
-        }
-
-        // Always check that our framebuffer is ok
-        if (glCheckFramebufferStatus(GL_FRAMEBUFFER) !=
-            GL_FRAMEBUFFER_COMPLETE) {
-            m_log->error() << "RenderManagerOpenGL::RenderEyeInitialize: Incomplete "
-                              "Framebuffer";
-            return false;
-        }
-
         // Call the display set-up callback for each eye, because they each
```

Comment:
What happens if the client changes the framebuffer in the rendering code?

Reply:
Going to revert this change.